src/builder/webpack/config.common.js

```diff
@@ -21,7 +22,7 @@ export default (config) => {
         ),
 
         PHENOMIC_USER_PATHNAME: wrap(process.env.PHENOMIC_USER_PATHNAME),
-
+        PHENOMIC_USER_URL: wrap(url.format(config.baseUrl)),
```

Comment:
can you add it into the doc as well? I documented other values https://github.com/MoOx/phenomic/blob/4dc38cffb8cc37cd1747297106907e2c2ce759e3/docs/content/docs/usage/scripting.md#constants